scripts/release

```diff
@@ -154,9 +154,13 @@ cut_release() {
     echo -e "\n## Component changes" | cat - "$rootdir/CHANGELOG.md" > /tmp/out && mv /tmp/out "$rootdir/CHANGELOG.md"
     cat "$CHANGELOG_PATH" | cat - "$rootdir/CHANGELOG.md" > /tmp/out && mv /tmp/out "$rootdir/CHANGELOG.md"
     echo -e "## API changes" | cat - "$rootdir/CHANGELOG.md" > /tmp/out && mv /tmp/out "$rootdir/CHANGELOG.md"
+    echo -e "Replace this text with example code for each new feature." | cat - "$rootdir/CHANGELOG.md" > /tmp/out && mv /tmp/out "$rootdir/CHANGELOG.md"
```

Comment:
Given the "don't block the release" guidance in the releasing docs, would not having these sections before the release is ready be a sufficient condition for rolling them back?  If the author of a change is OOO and there isn't example code ready to go, should we just roll-back that commit?

Reply:
That's a super good question. I'm inclined to say yes so that we encourage better commit descriptions.

Reply:
Great! As a release cop, I don't think I could write good example code for changes I don't know much about.